src/Ocelot/Buffering/BufferingMiddlewareExtensions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Builder;
+
+namespace Ocelot.Buffering;
+
+public static class BufferingMiddlewareExtensions
+{
+    public static IApplicationBuilder UseBufferingMiddleware(this IApplicationBuilder builder)
```

Comment:
Defining methods that are referenced only once throughout the entire solution is not advantageous. Moreover, since this middleware will be a system-level one, it can be incorporated into the pipeline using the `UseMiddleware<T>` method. Therefore, this definition does not address DRY issues.

Reply:
Just following the logic of `MultiplexingMiddlewareExtensions` (which in my opinion is not necessary too).

100% OK to remove this useless class.